tools/cli/go-whisk-cli/wski18n/resources/en_US.all.json

```diff
@@ -1412,8 +1412,8 @@
     "translation": "display full description of each API"
   },
   {
-    "id": "An API host must be provided.",
```

Comment:
This message is duplicated around line 1187 in this file. Not sure how we introduced the duplicate in the first place, but that's why we deleted it.